scripts/load-in-docker.sh

```diff
@@ -69,7 +69,7 @@ for name in "$gfile" "$Gfile" "$mfile" "$Mfile"; do
 			csvname=${name%.xlsx}.csv
 
 			# Convert to CSV if CSV is missing or old
-			# shellcheck disable=SC2039
+			# shellcheck disable=SC2039,SC3013
```

Comment:
and here?

Reply:
`-ot` is not a POSIX thing, but we still want to use it.